src/Raven.Client/Documents/Smuggler/DatabaseSmuggler.cs

```diff
@@ -127,6 +133,13 @@ private async Task<Operation> ExportAsync(DatabaseSmugglerExportOptions options,
                     getOperationIdCommand.NodeTag,
                     additionalTask);
             }
+            catch (Exception e)
+            {
+                if (requestTask == null)
+                    returnContext?.Dispose();
+
+                throw e.ExtractSingleInnerException();
```

Comment:
Why are you throwing it like this? We are losing the stack trace in this manner.

Reply:
It is the same in every place, probably to avoid throwing AggregateException